x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/firehose/progress_callout.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiText, EuiIconTip, EuiHorizontalRule } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { css } from '@emotion/react';
+import { ProgressIndicator } from '../shared/progress_indicator';
+
+export function ProgressCallout() {
+  return (
+    <ProgressIndicator
+      data-test-subj="observabilityOnboardingFirehoseProgressCallout"
+      title={
+        <EuiFlexGroup alignItems="center" gutterSize="s">
+          <EuiText>
+            <p>
+              {i18n.translate('xpack.observability_onboarding.firehosePanel.waitingForDataTitle', {
+                defaultMessage:
+                  'Retrieving data from Amazon Data Firehose... Detecting available services',
```

Comment:
I would stick to one message here. Having both adds unnecessary clutter.

Either "Retrieving data from Amazon Data Firehose..." or "Detecting available services..."